internal/otelcollector/config/metricgateway/types.go

```diff
@@ -0,0 +1,56 @@
+package metricgateway
+
+type KymaStatsReceiver struct {
+	AuthType           string      `yaml:"auth_type"`
+	CollectionInterval string      `yaml:"collection_interval"`
+	Resources          []ModuleGVR `yaml:"resources"`
+	K8sLeaderElector   string      `yaml:"k8s_leader_elector"`
+}
+
+type MetricConfig struct {
+	Enabled bool `yaml:"enabled"`
+}
+
+type ModuleGVR struct {
+	Group    string `yaml:"group"`
+	Version  string `yaml:"version"`
+	Resource string `yaml:"resource"`
+}
+
+// type Processors struct {
```

Comment:
If we don't need the commented `Processors` struct anymore, then we can delete it